README.md

```diff
@@ -154,6 +154,33 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Pushing Changes to a Remote Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to a remote Cluster for others to review.  This involves building a local image, pushing the image to Quay, then updating a remote cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. Quay Account <br> https://quay.io/signin/
```

Comment:
Quay is not necessarily required, just one option. You can use any image registry. I would say something like

```
2. An image registry like quay.io
```

and skip the details since they will be different depending on the registry.

Reply:
ok, I was going for a more of a tutorial.  I'll just link to those registries's 'getting started' pages.